src/executors/IntervalExecutor.js

```diff
@@ -4,6 +4,19 @@ import {getArgNames} from '../util/function';
 import {buildArgumentsFromRingaEvent} from '../util/executors';
 
 class IntervalExecutor extends ExecutorAbstract {
+  //-----------------------------------
+  // Constructor
+  //-----------------------------------
+  /**
+   * Constructor for a new IntervalExecutor.
```

Comment:
Might be good to clarify the internal mechanism this uses:

Constructor for a new IntervalExecutor, which internally uses the setInterval function.